tests/python/unittest/test_operator.py

```diff
@@ -2265,18 +2317,18 @@ def check_instance_norm_with_shape(shape, xpu):
                            numeric_eps=1e-2, rtol=1e-2, atol=1e-2)
 
 
+@with_seed()
 def test_instance_normalization():
     check_instance_norm_with_shape((1, 1, 1), default_context())
     check_instance_norm_with_shape((2, 1, 2), default_context())
     check_instance_norm_with_shape((2,4,5,6), default_context())
     check_instance_norm_with_shape((3,3,2,3,2,1,1), default_context())
 
 
-def check_l2_normalization(in_shape, mode, ctx=default_context(), norm_eps=1e-10):
+def check_l2_normalization(in_shape, mode, norm_eps=1e-10):
+    ctx = default_context()
     data = mx.symbol.Variable('data')
     out = mx.symbol.L2Normalization(data=data, mode=mode, eps=norm_eps)
-    # TODO(szha): Seeding this masks failures. We need to do a deep dive for failures without this seed.
```

Comment:
Don't remove the TODO

Reply:
I moved both the TODO and the seed-setting to the beginning of the test

Reply:
Ah I see, thanks